### PR TITLE
Update third-party-libraries.md

### PR DESCRIPTION
**Pull request by DawidWraga**
## Related Issues or Discussions

Fixes #

## Summary

Add 3rd party package link

## Check List

- [ ] `yarn run prettier` for formatting code and docs

### CODE REVIEW

**Review by dai-shi — CHANGES_REQUESTED** [review]
Please order it alphabetically.

**Comment by DawidWraga** [comment]
> Please order it alphabetically.

Thanks for letting me know, I have updated the order.

I have a lot of respect for all your work on Zustand, Valtio and Jotai, thank you for your contributions!

If you have a spare moment I'd love to hear your feedback on the [@davstack/store](https://www.npmjs.com/package/@davstack/storel) api design.

File: docs/integrations/third-party-libraries.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ This can be done using third-party libraries created by the community.
 > for someone looking to extend Zustand's feature set.
 
 - [@colorfy-software/zfy](https://colorfy-software.gitbook.io/zfy/) — 🧸 Useful helpers for state management in React with Zustand.
+- [@davstack/store](https://www.npmjs.com/package/@davstack/store) - A zustand store factory that auto generates selectors with get/set/use methods, supports inferred types, and makes global / local state management easy.
```
Comment (sewera) [inline docs/integrations/third-party-libraries.md:18]:
```suggestion
- [@davstack/store](https://www.npmjs.com/package/@davstack/store) — A zustand store factory that auto generates selectors with get/set/use methods, supports inferred types, and makes global / local state management easy.
```

**Review by dai-shi — APPROVED** [review]
Thanks!

**Pull request merged**